packages/core/src/components/Chip/Chip.css

```
@@ -0,0 +1,26 @@
+.root {
+  color: var(--color-greyLightest);
+  background: var(--color-greyDarkest);
+  font-family: var(--font-avenir);
+  padding: 0.375rem 0.625rem;
+  text-decoration: none;
+  display: inline-block;
+  height: auto;
+  width: auto;
+  border: 1px solid var(--color-greyDarkest);
+  box-sizing: border-box;
+  font-weight: 600;
```

We don't support 600 font weight in Avenir, it's 400, 700 and 800, should this be:
```suggestion
  font-weight: var(--fontweight-demi);
```

---

Yeah true, will double check with Oli about this.